app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectList.js

```diff
@@ -10,16 +10,39 @@ import { startCase } from 'lodash';
   to display a list of digital objects.
 */
 
+
+const storeSearchQueryParams =  (orderBy, totalCount, perPage, offset, pageNumber, q, filters) => {
+  if (q || filters.length > 0) {
+    window.sessionStorage.setItem('searchQueryParams',
+    JSON.stringify(
```

Comment:
We should set you up with in-IDE js linting because there a couple of js linter errors that I'm seeing when I run this branch in my local copy.  We're using eslint, which is like rubocop for javascript.  Our eslint settings come from this file: https://github.com/cul/ldpd-hyacinth/blob/master/.eslintrc